### ISSUE TITLE
As a new user I want the app to display my mnemonic seed on my profile so that I can see my seed phrase

### ISSUE THREAD

**Issue by JasminFarhi** [dltxio/idem-mobile#50]
- As a new user,
- When I use the app for the first time,
- Then Im given a 12 word seed on the profile screen. 

Save to local file system

Please remove this line once ticket is done. 
on app/src/navigation/screens/Home.tsx
Line 25 - console.log(mnemonic); <- remove

Front end is an alert when first loading the app (NOTE: NO CUSTOM MODAL)

**Comment by jrocca82** [comment]
This is already done on another ticket
